### PR TITLE
@dzucconi: Exclude artworks

### PR DESCRIPTION
**Pull request by dzucconi**
cc @mzikherman

### CODE REVIEW

**Comment by dzucconi** [comment]
Sort closes https://github.com/artsy/metaphysics/issues/200 would be nice to figure out a way to do this automagically.

File: schema/partner_show.js [hunk 1]
```diff
@@ -108,15 +111,20 @@ const PartnerShowType = new GraphQLObjectType({
...
       },
       resolve: (show, options) => {
         const path = `partner/${show.partner.id}/show/${show.id}/artworks`;
 
-        if (options.all) {
-          return gravity.all(path, options);
-        }
+        let fetch = null;
+        if (options.all) fetch = gravity.all(path, options);
+        fetch = gravity(path, options);
```
Comment (mzikherman) [inline schema/partner_show.js]: Confused, wouldn't `fetch` get overwritten even if `all` is passed in?

Comment (dzucconi) [inline schema/partner_show.js]:
```javascript
if (foo) bar = 'baz';
// is equivalent to
if (foo) {
  bar = 'baz';
}
```

Comment (dzucconi) [inline schema/partner_show.js]: Oh.

Damn. Good catch.

**Pull request merged**